src/modules/form-dyn-loading.js

```diff
@@ -0,0 +1,33 @@
+(function ($) {
+
+	'use strict';
+
+	var onSubmitting = function (key, data) {
+		window.updateState(data.ctn, true, 'loading');
+	};
+
+	var onComplete = function (key, data) {
+		window.updateState(data.ctn, false, 'loading');
```

Comment:
Should have a error/success state on the ctn in case we need to style it differently based on status.

Reply:
You don't know yet in the loading if it's gonna be an error or a success.

Reply:
I'm talking about the _complete_ state, not the _loading_.

Reply:
Yup.